### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

**Comment by kyrylo** [comment]
Let's add a changelog file prior to releasing this version. It would be nice to have that so that our users can keep track of the changes we're making.

**Comment by smurf-U** [comment]
> Let's add a changelog file prior to releasing this version. It would be nice to have that so that our users can keep track of the changes we're making.

In releases, I'm including changelogs like this. https://github.com/airbrake/pybrake/releases/tag/v1.2.0

**Comment by smurf-U** [comment]
PTAL

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+### Changed
+- Change naming style for remote_settings, settings_data
+
+### Fixed
+- remote_settings: fix fetching config route (The config route setting has no effect due to a bug where pybrake doesn't read that value prior to making a GET request to the notifier config server. The fix is to call `config_route()` on every notifier config GET call. Additionally, we are adding a logic that prevents remote settings to crash the background thread in case `config_route` is a bad value (HTTP lib would raise an error). When an error happens such as 403 Forbidden, we make a 2nd request to the old config route, which was known to work.)
+
+## [1.0.0] - 2021-01-21
+### Changed
+- Update year to match in every README example.
+- Reluctantly stop making fun of Brits only, since most of the world
```
Comment (kyrylo) [inline changelog.md:83]: 🤯⁇

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  writes dates in a strange way.
+
+### Fixed
+- Fix typos in recent README changes.
+- Update outdated unreleased diff link.
+
+
+[Unreleased]: https://github.com//airbrake/pybrake/compare/v1.3.0...HEAD
+[1.3.0]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.2.0...v1.3.0
```
Comment (kyrylo) [inline changelog.md:92]: Broken links

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- Update year to match in every README example.
+- Reluctantly stop making fun of Brits only, since most of the world
+  writes dates in a strange way.
+
+### Fixed
+- Fix typos in recent README changes.
+- Update outdated unreleased diff link.
+
+
+[Unreleased]: https://github.com//airbrake/pybrake/compare/v1.3.0...HEAD
```
Comment (kyrylo) [inline changelog.md:91]:
```suggestion
[Unreleased]: https://github.com/airbrake/pybrake/compare/v1.3.0...HEAD
```

File: changelog.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+## [1.0.2] - 2021-02-16
+### Removed
+- remote_settings: delete print that's left from debugging
+
+## [1.0.1] - 2021-02-16
+### Changed
+- Change naming style for remote_settings, settings_data
+
+### Fixed
+- remote_settings: fix fetching config route (The config route setting has no effect due to a bug where pybrake doesn't read that value prior to making a GET request to the notifier config server. The fix is to call `config_route()` on every notifier config GET call. Additionally, we are adding a logic that prevents remote settings to crash the background thread in case `config_route` is a bad value (HTTP lib would raise an error). When an error happens such as 403 Forbidden, we make a 2nd request to the old config route, which was known to work.)
```
Comment (kyrylo) [inline changelog.md:78]: Let's keep formatting at 80 chars per line. This way people can use different editors to read the changelog.

File: changelog.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+### Fixed
+- Fix query stats send to the Airbrake
+
+## [1.0.6] - 2022-04-13
+### Changed
+- Improve pytest
+
+### Fixed
+- [#183](https://github.com/airbrake/pybrake/issues/183) on Flask request filter
```
Comment (kyrylo) [inline changelog.md:38]: This looks confusing and not useful if read this changelog anywhere else but GitHub. Let's describe #183 here.

**Comment by smurf-U** [comment]
PTAL

File: changelog.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+is a bad value (HTTP lib would raise an error). When an error happens such as
+403 Forbidden, we make a 2nd request to the old config route, which was known
+to work.)
+
+## [1.0.0] - 2021-01-21
+
+### Added
+
+- Pybrake SDK
+- Middleware for Flask, Django, celery, aiohttop frameworks
```
Comment (kyrylo) [inline changelog.md:120]:
```suggestion
- Middleware for Flask, Django, celery, aiohttp frameworks
```

File: changelog.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+### Fixed
+
+- remote_settings: fix fetching config route (The config route setting has no 
+effect due to a bug where pybrake doesn't read that value prior to making a 
+GET request to the notifier config server. The fix is to call `config_route()`
+on every notifier config GET call. Additionally, we are adding a logic that 
+prevents remote settings to crash the background thread in case `config_route` 
+is a bad value (HTTP lib would raise an error). When an error happens such as
+403 Forbidden, we make a 2nd request to the old config route, which was known
+to work.)
```
Comment (kyrylo) [inline changelog.md:113]:
```suggestion
to work)
```

File: changelog.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+## [1.2.0] - 2022-05-11
+
+### Added
+
+- Bottle middleware
+- Bottle sample example
+
+### Fixed
+
+- Errors filtered out when middleware involve in backtrace [#191](https://github.com/airbrake/pybrake/issues/191)
```
Comment (kyrylo) [inline changelog.md:28]:
```suggestion
- Errors are filtered out when middleware is involved in backtrace [#191](https://github.com/airbrake/pybrake/issues/191)
```

**Review by kyrylo — APPROVED** [review]

**Comment by smurf-U** [comment]
PTAL

**Review by kyrylo — APPROVED** [review]

**Pull request merged**